pkg/provision/primitives/container.go

```diff
@@ -98,6 +98,10 @@ func (p *Provisioner) containerProvisionImpl(ctx context.Context, reservation *p
 		return ContainerResult{}, err
 	}
 
+	if config.Capacity.DiskType == pkg.HDDDevice {
```

Comment:
Actually since now we only support one type, should we keep the DiskType field at all ?

Reply:
Same as above